cg/services/illumina/backup/backup_service.py

```diff
@@ -278,37 +286,76 @@ def retrieve_archived_file(self, archived_file: Path, run_dir: Path) -> None:
             file_path=str(archived_file), target_path=str(retrieved_file)
         )
 
+    @staticmethod
+    def parse_dsmc_output_sequencing_path(dsmc_output: list[str]) -> list[DsmcSequencingFile]:
+        """Parses the DSMC command output to extract validated sequencing paths."""
+        validated_responses = []
+        for line in dsmc_output:
+            if is_dsmc_sequencing_path(line):
+                parts = line.split()
+
+                fileDateTime: datetime = convert_string_to_datetime_object(
+                    f"{parts[DsmcOutput.DATE_COLUMN_INDEX]} {parts[DsmcOutput.TIME_COLUMN_INDEX]}"
+                )
+
+                query_response = DsmcSequencingFile(
+                    dateTime=fileDateTime,
+                    path=Path(parts[DsmcOutput.PATH_COLUMN_INDEX]),
+                )
+                validated_responses.append(query_response)
+
+        return validated_responses
+
     @classmethod
-    def get_archived_sequencing_run_path(cls, dsmc_output: list[str]) -> Path | None:
+    def get_latest_archived_sequencing_run_path(cls, dsmc_output: list[str]) -> Path | None:
```

Comment:
Hi @ChrOertlin! Would we like to return `None` if the sequencing run path is not found? Why is that? :) Are we testing if there is an archived flowcell?